### ISSUE TITLE
Adapt Postgres database driver

### ISSUE THREAD

**Issue by manyuanrong** [manyuanrong/dso#2]
Posgres database should be supported.
But wait until Deno's dynamic feature is complete.

**Comment by bartlomieju** [comment]
@manyuanrong let me know if you want to work on that, with joint effort we should be able to pull this off!

**Comment by manyuanrong** [comment]
@bartlomieju Yes, I want to have this feature. I haven't got the energy to start, but I've made a list.
#4
